package.json

```diff
@@ -62,12 +62,50 @@
           "orders.item.get",
           "orders.po-lines.collection.get",
           "orders.po-lines.item.get",
-          "orders.receiving-history.collection.get"
+          "orders.receiving-history.collection.get",
+          "orders.po-number.item.get"
+        ]
+      },
+      {
+        "permissionName": "ui-orders.orders.modify",
+        "displayName": "Orders: permissions to view, create and edit order and lines with receiving and check-in",
+        "description": "",
+        "visible": true,
+        "subPermissions": [
+          "ui-orders.view",
+          "orders.item.post",
+          "orders.item.put",
+          "orders.po-number.item.post",
+          "orders.po-lines.item.post",
+          "orders.po-lines.item.put",
+          "orders.receiving.collection.post",
+          "orders.check-in.collection.post",
+          "orders.pieces.item.post"
+        ]
+      },
+      {
+        "permissionName": "ui-orders.order.delete",
+        "displayName": "Orders: Remove orders",
+        "description": "",
+        "visible": true,
+        "subPermissions": [
+          "ui-orders.view",
```

Comment:
I'm not so sure about this. Basically, just map ui permission to back-end should be enough, since there is a separate view permission.

Reply:
it's just to be sure that users only with this permission will be able to open views without errors
(we don't know how users will be configured)

but I'm ok to remove this if you think it's redundant

Reply:
There a lot of unknown variables then, let's stick to your vision, it looks safer